upstream-community-operators/prisma-cloud-compute-console-operator/1.0.0/prisma-cloud-compute-console-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,338 @@
+    certified: 'false'
+    containerImage: prismacloud/compute-console-operator:1.0.0
+    createdAt: '2019-04-01 00:00:00'
+    description: Deploy Prisma Cloud Compute (Twistlock) cloud native security in Kubernetes.
+    repository: https://github.com/twistlock/sample-code/tree/master/operators/prisma-cloud-compute-console-helm-operator
+    support: Palo Alto Networks
+  name: prisma-cloud-compute-console-operator.v1.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+      - kind: TwistlockConsole
+        name: twistlockconsoles.charts.helm.k8s.io
+        version: v1alpha1
+        displayName: Twistlock
+        description: |
+          Twistlock Console is installed first and provides policy, API
+          endpoints, GUI, and makes install of Defenders on each node easy
+          through a daemonset.
+        resources:
+        - kind: ReplicationController
+          name: ''
+          version: v1
+        - kind: Service
+          name: ''
+          version: v1
+        - kind: PersistentVolumeClaim
+          name: ''
+          version: v1
+        - kind: ConfigMap
+          name: ''
+          version: v1
+        - kind: Secret
+          name: ''
+          version: v1
+        - Kind: ServiceAccount
+          name: ''
+          version: v1
+  displayName: Prisma Cloud Compute (Twistlock) Console Operator
+  icon:
+    - base64data: >-
+        PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAzNy44MyA0MC42Ij48ZGVmcz48c3R5bGU+LmNscy0xe2ZpbGw6IzAxMDEwMTt9PC9zdHlsZT48L2RlZnM+PHRpdGxlPlByaXNtYS1jbG91ZC1ibGFjazwvdGl0bGU+PHBvbHlnb24gY2xhc3M9ImNscy0xIiBwb2ludHM9IjIwLjMgMjcuMDcgMjAuMyA4LjI3IDMwLjEyIDI3LjA3IDIwLjMgMjcuMDciLz48cG9seWdvbiBjbGFzcz0iY2xzLTEiIHBvaW50cz0iMjAuMyA4LjI3IDEwLjQ4IDI3LjA3IDIwLjMgMjcuMDcgMjAuMyA0MC42IDAgMjAuMyAxMC4zOCA5LjkyIDEyLjAzIDguMjcgMjAuMyAwIDIwLjMgOC4yNyIvPjxwYXRoIGNsYXNzPSJjbHMtMSIgZD0iTTM3LjgzLDI3LjA3YTYuMDYsNi4wNiwwLDEsMSwwLTEyLjExdjIuNGEzLjY1LDMuNjUsMCwxLDAsMCw3LjNaIi8+PC9zdmc+
+      mediatype: image/svg+xml
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: false
+      type: AllNamespaces
+  keywords:
+    - twistlock
+    - prisma
+    - prisma-cloud
+    - prisma-cloud-compute
+    - paloalto 
+    - paloaltonetworks
+    - security
+    - monitoring
+    - scanning
+    - runtime-security
+    - container security
+    - kuberenetes security
+    - openshift security
+    - container vulnerabilities
+    - cloud native application security
+    - compute
+  links:
+    - name: Source Code
+      url: https://github.com/twistlock/sample-code/tree/master/operators/prisma-cloud-compute-console-helm-operator
+  maturity: beta
+  provider:
+    name: Palo Alto Networks
+  replaces: twistlock-console-operator.v0.0.9
```

Comment:
@gunjan5, replaces is used to replace older version of operator wit a new release. Correct me if I'm wrong.
There are 2 operators within one directory and Prisma is replacing Twistlock. This PR is adding new Prisma operator to catalog. Not sure if it works. need to double check.